imports/ui/styles/dark.scss

```diff
@@ -191,6 +192,10 @@ $accordion-bg:                          $grey;
 .dt-scrollable__no-data {
     border:                 0px !important;
 }
+.d-flex {
```

Comment:
please remove this, I don't see where it is needed, d-flex is defined in bootstrap core and the defining the color on this level is causing more issues than it solves.